documentation/faq/70-packages.md

```diff
@@ -2,10 +2,14 @@
 question: How do I fix the error I'm getting trying to include a package?
 ---
 
-SSR in Vite is not yet stable. Libraries work best with Vite when they distribute both CJS and ESM in their package and you may wish to work with library authors to make this happen.
+Vite's SSR support has become fairly stable since Vite 2.7. Most issues related to including a library are due to incorrect packaging.
 
-Svelte components must be written entirely in ESM. It is encouraged to make sure the dependencies of external Svelte components provide an ESM version. However, in order to handle CJS dependencies [`vite-plugin-svelte` will look for any CJS dependencies of external Svelte components](https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#what-is-going-on-with-vite-and-pre-bundling-dependencies) and ask Vite to pre-bundle them by automatically adding them to Vite's `optimizeDeps.include` which will use `esbuild` to convert them to ESM.
+Libraries work best with Vite when they distribute an ESM version and you may wish to suggest this library authors. Here's a few things to keep in mind when checking if a library is packaged correctly:
+- `exports` takes precedence over the other fields like `main`, `module`, and `browser`. Adding an `exports` field may not be backwards-compatible as it prevents deep imports.
+- `main` should be defined if `exports` is not. It should be a CommonJS file unless `"type": "module"` is set, in which case it should be an ESM file. If a `module` field is defined, it should refer to an ESM file.
+- ESM files should end with `.mjs` unless `"type": "module"` is set in which any case CommonJS files should end with `.cjs`
+- Svelte components should be distributed entirely as ESM and have a `svelte` field defining the entry point
```

Comment:
I thought we were planning to deprecated the `svelte` field. I remember rich talking about it.

Reply:
there was some discussion in the Kit repo (e.g. could it be a boolean rather than a path), but no decisions were made about changing anything yet, and so I think for now it's still the recommended course of action to have it